test/runtime/samples/await-then-catch-in-slot/_config.js

```diff
@@ -29,7 +29,7 @@ export default {
 
 		try {
 			await deferred.promise;
-		} catch {}
+		} catch {} // eslint-disable-line no-empty
```

Comment:
this will get fixed when switching to my lint config, so i'll remove it here to avoid needing to do it in the next PR
```suggestion
		} catch {}
```